### DISCUSSION TITLE
Show Member and Collaborator badges in the comments

### DISCUSSION THREAD

**Discussion by Jackenmen** [community/community#9453] [Mobile]
Currently, you can't easily determine if someone is organisation member or repository collaborator without going to web version of the issue/PR/discussion *and* choosing desktop version of it.
It would be great if it was easier to figure out if someone is more than just a regular user so that I know if this person speaks as a maintainer or is just a random person who doesn't represent the project.

**Comment by brianlovin** [comment]
Hey @jack1142 which platform are you on, iOS or Android? We should be showing contributor labels underneath usernames on comments to make it more clear what a person's role is I a repo, but it's possible we're missing a badge in certain locations. If you have an example of a URL (public) that we could look at, it would be helpful!

**Reply by Jackenmen** [reply]
Android - GitHub Mobile v1.45.0-beta (10040)

Here are some example links:
https://github.com/solokeys/solo/issues/208#issuecomment-1003258497
![Screenshot_20220105-014040_GitHub.jpg](https://user-images.githubusercontent.com/6032823/148143029-07e139be-2a0b-4dac-a788-c9326630a47a.jpg)

https://github.com/github/feedback/discussions/9453#discussioncomment-1909177
![Screenshot_20220105-014349_GitHub.jpg](https://user-images.githubusercontent.com/6032823/148143045-af2a6aa3-6fc3-4886-b14c-de89d96410f0.jpg)

**Reply by brianlovin** [reply]
Thank you! I can reproduce some missing/incorrect badges on my side, especially in Discussions. I'll do an audit and pass this along to our team.
